ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -88,74 +98,98 @@ const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
-          />
-        </View>
-      </React.Fragment>
-    );
-  }
-
-  public render(): React.ReactNode {
-    const selectedWallet = this.props.navigation.getParam("selectedWallet");
-
-    const isFavorite = pot.map(
-      this.props.favoriteWallet,
-      _ => _ === selectedWallet.idWallet
-    );
-
-    // to retro-compatibility purpose
-    const pm = pot.getOrElse(
-      pot.map(this.props.paymentMethods, pms =>
-        pms.find(pm => pm.idWallet === selectedWallet.idWallet)
-      ),
-      undefined
-    );
-
-    return (
-      <WalletLayout
-        title={I18n.t("wallet.paymentMethod")}
-        allowGoBack={true}
-        topContent={this.headerContent(selectedWallet, isFavorite)}
-        hideHeader={true}
-        hasDynamicSubHeader={true}
-        topContentHeight={HEADER_HEIGHT}
-        contextualHelpMarkdown={contextualHelpMarkdown}
-        faqCategories={["wallet_transaction"]}
-      >
-        {pm && (
-          <>
-            <View style={IOStyles.horizontalContentPadding}>
-              <View spacer={true} extralarge={true} />
-              <PaymentMethodCapabilities paymentMethod={pm} />
-              <View spacer={true} />
-              <ItemSeparatorComponent noPadded={true} />
-            </View>
-            <EdgeBorderComponent />
-          </>
-        )}
-      </WalletLayout>
-    );
-  }
-}
+const headerContent = (
+  selectedWallet: Wallet,
+  isFavorite: pot.Pot<boolean, Error>,
+  setFavorite: (walletId?: number) => void,
+  onDelete: (walletId: number) => void
+) => (
+  <>
+    <View style={styles.cardBox}>
+      <CardComponent
+        type={"Header"}
+        wallet={selectedWallet}
+        hideFavoriteIcon={false}
+        hideMenu={false}
+        isFavorite={isFavorite}
+        onSetFavorite={(willBeFavorite: boolean) =>
+          handleSetFavourite(willBeFavorite, () =>
+            setFavorite(selectedWallet.idWallet)
+          )
+        }
+        onDelete={() => onDelete(selectedWallet.idWallet)}
+      />
+    </View>
+  </>
+);
+
+const TransactionsScreen: React.FC<Props> = (props: Props) => {
+  const selectedWallet = props.navigation.getParam("selectedWallet");
+
+  const isFavorite = pot.map(
+    props.favoriteWallet,
+    _ => _ === selectedWallet.idWallet
+  );
+
+  // to retro-compatibility purpose
+  const pm = pot.getOrElse(
+    pot.map(props.paymentMethods, pms =>
+      pms.find(pm => pm.idWallet === selectedWallet.idWallet)
+    ),
+    undefined
+  );
+  const { present } = useRemovePaymentMethodBottomSheet({
+    icon: selectedWallet.creditCard
+      ? getCardIconFromBrandLogo(selectedWallet.creditCard)
+      : defaultCardIcon,
+    caption:
+      selectedWallet.paymentMethod &&
+      isRawCreditCard(selectedWallet.paymentMethod)
+        ? getTitleFromCard(selectedWallet.paymentMethod)
+        : FOUR_UNICODE_CIRCLES
+  });
+
+  const UnsubscribeButton = (props: { onPress?: () => void }) => (
```

Comment:
```suggestion
  const DeletePaymentMethodButton = (props: { onPress?: () => void }) => (
```
Is more descriptive?

Reply:
Sounds good, updated in [fc2232c](https://github.com/pagopa/io-app/pull/2917/commits/fc2232cc67ccd0a8ca7e6266b1c902046cc2b3d7)